doc/source/basics.rst

```diff
@@ -1280,12 +1280,29 @@ You can also chain these types of operations:
 
    s.dt.tz_localize('UTC').dt.tz_convert('US/Eastern')
 
+You can also format datetime values as strings with ``Series.dt.strftime`` which
```

Comment:
although that is maybe not that important here as it is exactly the same as standard library